src/main/catalog-sources/__test__/kubeconfig-sync.test.ts

```diff
@@ -46,16 +48,18 @@ describe("kubeconfig-sync.source tests", () => {
 
     clusters = new Map();
     di.override(getClusterByIdInjectable, () => id => clusters.get(id));
-    di.override(directoryForUserDataInjectable, () => ({
-      get: () => "some-directory-for-user-data",
-    }));
```

Comment:
Now that I think of it, getting rid of all these boilerplate overrides for directories is good. The stubbed directories of global overrides are well enough to communicate all relevancy in this.